.editorconfig

```diff
@@ -0,0 +1,12 @@
+# This file is for unifying the coding style for different editors and IDEs
+# editorconfig.org
+
+root = true
```

Comment:
Could you remove this file please? Or even better, add it to the .gitignore file?
Although this is nice of course, it is very specific for a (set of) editors, whilst ignoring others. Next to that, I'd much rather have these rules enforced with something like grumphp and php-cf-fixer, so it doesn't matter which editor people use.

Reply:
You're right. It can use the ```php-cs-fixer``` or [StyleCI](https://styleci.io) and let the command in the Composer script.
Would you prefer which one to replace the ```.editorconfig```?

Thanks.